pycsob/client.py

```diff
@@ -71,9 +68,10 @@ def payment_init(self, order_no, total_amount, return_url, description, cart=Non
         :param total_amount:
         :param return_url: URL to be returned to from payment gateway
         :param cart: items in cart, currently min one item, max two as mentioned in CSOB spec
-        :param description: order description
+        :param description: product name
```

Comment:
I would add that it's only used for the cart item name if cart is emtpy.

Reply:
There should be changelog entry with incompatible and other changes.

Reply:
Docstring and Changelog  have been updated.